### DISCUSSION TITLE
关于公式环境里符号字体的问题

### DISCUSSION THREAD

**Discussion by ZehuaJia** [sjtug/SJTUThesis#848] [提问] [answered]
Hello,各位好，我在编译的时候发现一个关于公式环境下符号字体的问题。
（我用的是Tex Live 2021）
同样一段公式，在日常英文小论文写作时(PDFLaTex编译)，常见的显示如下：
![5a38ea7db6706d036235ea274c453c5](https://user-images.githubusercontent.com/43306569/194342433-c089f981-2d30-46d6-8705-5a12d9ce1e57.png)

但是在模板里的显示如下（XeLaTex编译）：
![af63aa94660e3a46da29a5372d70f8c](https://user-images.githubusercontent.com/43306569/194341452-4d263cf3-881b-4bfd-81ec-8ed4e5c25b14.png)
二者字体是不相同的，想请问一下如何才能改成上面版本的字体呢？
谢谢！

**Comment by LogCreative** [comment, marked as answer]
```latex
\documentclass[math-font=newcm]{sjtuthesis}
```

可以在 [`fd 文件夹`](https://github.com/sjtug/SJTUThesis/tree/master/texmf/tex/latex/sjtuthesis/fd) 中查看其他字体组合，他们对应着 `cjk-font`，`math-font`，`text-font` 选项。

**Reply by ZehuaJia** [reply]
感谢，选择`\documentclass[math-font=lm]{sjtuthesis}` 成功解决。
